README.md

```diff
@@ -82,4 +82,7 @@ Here are some SQL queries for most common use cases
 
 ![SQL Query1](resources/screenshots/query1.png)
 
-![SQL Query2](resources/screenshots/query2.png)
\ No newline at end of file
+![SQL Query2](resources/screenshots/query2.png)
+
+### DBT dag diagram
```

Comment:
Style: I'd suggest putting the images closer to the top of the readme (visuals are good!).
For this section, a better title could be "Lineage Graph".